.github/workflows/acceptance.yml

```diff
@@ -31,7 +31,8 @@ jobs:
     # Map a step output to a job output
     outputs:
       acceptance: ${{ steps.changes.outputs.acceptance }}
-      backend: ${{ steps.changes.outputs.backend_all }}
+      backend_all: ${{ steps.changes.outputs.backend_all }}
```

Comment:
You can see that we were condition on `backend_all` but only `backend` would be available before this.
https://github.com/getsentry/sentry/blob/b9ed360540ae6d31bdc827a500ec8f761ead5fd4/.github/workflows/acceptance.yml#L215
https://github.com/getsentry/sentry/blob/b9ed360540ae6d31bdc827a500ec8f761ead5fd4/.github/workflows/acceptance.yml#L289